lib/client.ml

```diff
@@ -1,10 +1,24 @@
 open Lwt_unix
 
+module Message = struct
```

Comment:
Should it be extracted to a dedicated module?

Reply:
Yes